book/src/install.md

```diff
@@ -1,179 +1,221 @@
 
-```cmd
-cd %appdata%\helix
-mklink /D runtime "<helix-repo>\runtime"
+```sh
+ln -s $PWD/runtime ~/.config/helix/runtime
 ```
 
-The runtime location can be overridden via the `HELIX_RUNTIME` environment variable.
-
-> NOTE: if `HELIX_RUNTIME` is set prior to calling `cargo install --path helix-term --locked`,
-> tree-sitter grammars will be built in `$HELIX_RUNTIME/grammars`.
-
-If you plan on keeping the repo locally, an alternative to copying/symlinking
-runtime files is to set `HELIX_RUNTIME=/path/to/helix/runtime`
-(`HELIX_RUNTIME=$PWD/runtime` if you're in the helix repo directory).
+### Configuring Helix's runtime files for Windows
 
-To use Helix in desktop environments that supports [XDG desktop menu](https://specifications.freedesktop.org/menu-spec/menu-spec-latest.html), including Gnome and KDE, copy the provided `.desktop` file to the correct folder:
+Either set the `HELIX_RUNTIME` environment variable to point to the runtime files using the Windows setting (search for
+`Edit environment variables for your account`) or use the `setx` command in
+Cmd:
 
-```bash
-cp contrib/Helix.desktop ~/.local/share/applications
+```sh
+setx HELIX_RUNTIME "%userprofile%\source\repos\helix\runtime"
 ```
 
-To use another terminal than the default, you will need to modify the `.desktop` file. For example, to use `kitty`:
+> 💡 `%userprofile%` resolves to your user directory like
+> `C:\Users\Your-Name\` for example.
 
-```bash
-sed -i "s|Exec=hx %F|Exec=kitty hx %F|g" ~/.local/share/applications/Helix.desktop
-sed -i "s|Terminal=true|Terminal=false|g" ~/.local/share/applications/Helix.desktop
-```
+Or, create a symlink in `%appdata%\helix\` that links to the source code
+   directory:
 
-Please note: there is no icon for Helix yet, so the system default will be used.
+   | Method     | Command                                                                                |
+   | ---------- | -------------------------------------------------------------------------------------- |
+   | PowerShell | `New-Item -ItemType SymbolicLink -Target "runtime" -Path "$Env:AppData\helix\runtime"` |
+   | Cmd        | `cd %appdata%\helix` <br/> `mklink /D runtime "%userprofile%\src\helix\runtime"`       |
 
-## Finishing up the installation
+   > 💡 On Windows, creating a symbolic link may require running PowerShell or
+   > Cmd as an administrator.
 
-To make sure everything is set up as expected you should finally run the helix healthcheck via
+## Validating the Installation
 
-```
+To make sure everything is set up as expected you should run the Helix health
+check:
+
+```sh
 hx --health
 ```
 
-For more information on the information displayed in the health check results refer to [Healthcheck](https://github.com/helix-editor/helix/wiki/Healthcheck).
+For more information on the health check results refer to
+[Health check](https://github.com/helix-editor/helix/wiki/Healthcheck).
 
-### Building tree-sitter grammars
+## Configure the Desktop Shortcut
```

Comment:
Both of these subheadings could nested be under build from source, since the installation should be correct via a package manager (and we also assume they correctly install the desktop shortcut for you)

Reply:
Yes, I don't know why I didn't think of that :)